Enterprise-CMCS/macpro-onemac
# Reverting the change: Removing the MMDL cards on new chip/spa

Story: https://jiraent.cms.gov/browse/OY2-31696
Endpoint: https://d3nmqq9h68oslv.cloudfront.net/

### Details
 - added back MMDL card to Medicaid SPA:
<img width="1132" alt="Screenshot 2024-12-11 at 11 54 17 AM" src="https://github.com/user-attachments/assets/b15f3cfb-7580-4bd3-955a-56a2c4f0e18b" />
- with landing page when clicked:
<img width="995" alt="Screenshot 2024-12-11 at 11 56 18 AM" src="https://github.com/user-attachments/assets/4b28d9bc-a810-49c5-9ec9-8854719560ed" />
- added back MMDL card for CHIP:
<img width="903" alt="Screenshot 2024-12-11 at 11 54 28 AM" src="https://github.com/user-attachments/assets/2d4a319e-181e-45bb-be0c-2af41de04d51" />
- which directs to this landing page: 
<img width="943" alt="Screenshot 2024-12-11 at 11 55 48 AM" src="https://github.com/user-attachments/assets/ad20de93-dafc-4fb6-82a0-c1307d9d158d" />

### Changes

- added back landing pages
- added back routes to those pages
- added back card which allowed clicking to the card

### Implementation Notes
- since we were told to pause work on MMDL and this was the only MMDL work not wrapped with LaunchDarkly, we are adding it back in for now as a temporary fix  
- In the future this feat should be wrapped in LaunchDarkly 

### Test Plan
1. Click new submission
2. Click CHIP or SPA
3. Ensure the MMDL card exsists
4. Make sure the landing page is correct
